javascript/javascript1/week2/homework/main-2.js

```diff
@@ -0,0 +1,13 @@
+//EVENT APPLICATION
+function getEventWeekday(daysFromToday) {
+    const weekdays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]; //array with weekdays
+    const day = new Date();
+    let todaysDay = day.getDay(); //js method which returns eact today's day by it's number begins from 0 Sunday
+    let eventDay = todaysDay + daysFromToday; //calculates number of event day
+    let weekdayOfEvent = eventDay % 7; //transform number of the event day from 0 to 6
```

Comment:
These can all be `const`s. You will find you will hardly ever need to use `let`s now that you're writing functions :)